packaging/images/openshift-ci/Dockerfile.test-runtime

```diff
@@ -1,13 +1,15 @@
 FROM registry.access.redhat.com/ubi8/ubi:latest
 USER root
-RUN echo -e '[google-cloud-sdk]\n\
-name=Google Cloud SDK\n\
-baseurl=https://packages.cloud.google.com/yum/repos/cloud-sdk-el8-x86_64\n\
-enabled=1\n\
-gpgcheck=1\n\
-repo_gpgcheck=1\n\
-gpgkey=https://packages.cloud.google.com/yum/doc/yum-key.gpg\n\
-      https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg' > /etc/yum.repos.d/google-cloud-sdk.repo
+RUN printf '%s\n' \
```

Comment:
Does this produce the same output? It seems like we would lose the line breaks between each of the separate values?

Could we put this config file in git as a file and then copy it into the right place in the container at this step?

Reply:
TIL

```
$ printf "%s\n" foo bar
foo
bar
```